package.json

```diff
@@ -6,7 +6,8 @@
     "postinstall": "patch-package && rn-nodeify --install stream,crypto,fs,path --hack",
     "link": "react-native link",
     "android": "react-native run-android --appIdSuffix debug",
-    "ios": "react-native run-ios",
+    "ios": "react-native run-ios --simulator=\"iPhone 13 Pro\"",
+    "iphone": "react-native run-ios --device=\"iPhone di Alessandro\"",
```

Comment:
I believe these should not be pushed publicly since they are your own configurations @ale-mazz